<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>200</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stereo Calibration</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonCalibrate">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture frames</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxSampleCount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of images:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay between images</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxInterval">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Board width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Board height</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCornersX">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCornersY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="value">
     <number>7</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibration File:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFilename">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLoad">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSave">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSampleTimeout">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time left (in seconds):</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="calibrateCaera">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibrate cameras</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCalibrate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startCapture()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startCapture()</slot>
 </slots>
</ui>
